### PR TITLE
[keymap] Add abnt2 format to gh60

### PR DESCRIPTION
**Pull request by luizcorreia**
Added brazilian abnt2 format to gh60

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gh60/revc/keymaps/default_abnt2/keymap.c [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    _______, KC_LEFT, KC_DOWN, KC_RGHT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    KC_LCTL, KC_LGUI, KC_LALT,                            XXXXXXX,                            XXXXXXX, XXXXXXX, _______, XXXXXXX),
+
+};
+
+void matrix_scan_user(void) {
+
+//Layer LED indicators
+    uint32_t layer = layer_state;
```
Comment (drashna) [inline keyboards/gh60/revc/keymaps/default_abnt2/keymap.c]: No need to do this.

File: keyboards/gh60/revc/keymaps/default_abnt2/keymap.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    if (layer & (1<<1)) {
```
Comment (drashna) [inline keyboards/gh60/revc/keymaps/default_abnt2/keymap.c]: You could just use: 
```suggestion
    if (layer_state_is(_FL)) {
```

File: keyboards/gh60/revc/keymaps/default_abnt2/keymap.c [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        gh60_wasd_leds_on();
+        gh60_fn_led_on();
+    } else {
+        gh60_wasd_leds_off();
+        gh60_fn_led_off();
+    }
+
+    if (layer & (1<<2)) {
```
Comment (drashna) [inline keyboards/gh60/revc/keymaps/default_abnt2/keymap.c]:
```suggestion
    if (layer_state_is(_CL)) {
```

File: keyboards/dz60/keymaps/default_abnt2/readme.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # BR ABNT2 layout
 
-This is a Brazilian ABNT2 keymap for the DZ60 in a standard layout.
+This is a Brazilian ABNT2 keymap for the dz60 in a standard layout.
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/default_abnt2/readme.md]: This change should be reverted.

```suggestion
This is a Brazilian ABNT2 keymap for the DZ60 in a standard layout.
```

File: keyboards/gh60/revc/info.json [hunk 5]
```diff
@@ -11,414 +11,2339 @@
...
-                {"label":"HHKB Fn", "x":14, "y":3},
-                {"label":"Ctrl", "x":0, "y":4, "w":1.25},
-                {"label":"Win", "x":1.25, "y":4, "w":1.25},
-                {"label":"Alt", "x":2.5, "y":4, "w":1.25},
-                {"label":"Space", "x":3.75, "y":4, "w":6.25},
-                {"label":"Alt", "x":10, "y":4, "w":1.25},
-                {"label":"Win", "x":11.25, "y":4, "w":1.25},
-                {"label":"Menu", "x":12.5, "y":4, "w":1.25},
-                {"label":"Ctrl", "x":13.75, "y":4, "w":1.25}
+                {
```
Comment (noroadsleft) [inline keyboards/gh60/revc/info.json]: I would much prefer if this file kept the existing formatting of one key object per line and one line per object.

Comment (luizcorreia) [reply keyboards/gh60/revc/info.json]: Me too, I just follow the file configuration.

**Comment by luizcorreia** [comment]
@noroadsleft The abnt2 format is the default used on all keyboards in Brazil, it differs from iso in having one more key splinting the right shift. 
I didn't quite understand the community layout.

**Comment by noroadsleft** [comment]
@luizcorreia,

Community Layouts is a feature in QMK Firmware that allows a user to write one keymap that can be used on multiple keyboards, if the user owns multiple keyboards that share the same physical layout, such as owning a DZ60, a GH60 rev. C and a Zeal60, all built with a standard ANSI layout.

Instead of having something like a `luizcorreia` keymap in each of `keyboards/dz60/`, `keyboards/gh60/revc/` and `keyboards/zeal60/`, you could write one `keymap.c` that goes in `layouts/community/60_abnt2/luizcorreia/` and use that with any keyboard that's configured to support the `60_abnt2` layout.

**Comment by luizcorreia** [comment]
> @luizcorreia,
> 
> Community Layouts is a feature in QMK Firmware that allows a user to write one keymap that can be used on multiple keyboards, if the user owns multiple keyboards that share the same physical layout, such as owning a DZ60, a GH60 rev. C and a Zeal60, all built with a standard ANSI layout.
> 
> Instead of having something like a `luizcorreia` keymap in each of `keyboards/dz60/`, `keyboards/gh60/revc/` and `keyboards/zeal60/`, you could write one `keymap.c` that goes in `layouts/community/60_abnt2/luizcorreia/` and use that with any keyboard that's configured to support the `60_abnt2` layout.

I see, but I don't want a layout with my name, I want help people in Brazil, that want a keyboard with abnt2 format. I will make a community layout in this PR or create another?

**Comment by noroadsleft** [comment]
> I see, but I don't want a layout with my name, I want help people in Brazil, that want a keyboard with abnt2 format. I will make a community layout in this PR or create another?

It would be done in another PR. I could actually define the Community Layout, which would dictate the order any supporting keyboards would need to use, though the keyboards would still need to be modified as you've done here.

I'll make a PR and tag you so you can see what I'm talking about.

**Comment by luizcorreia** [comment]
> > I see, but I don't want a layout with my name, I want help people in Brazil, that want a keyboard with abnt2 format. I will make a community layout in this PR or create another?
> 
> It would be done in another PR. I could actually define the Community Layout, which would dictate the order any supporting keyboards would need to use, though the keyboards would still need to be modified as you've done here.
> 
> I'll make a PR and tag you so you can see what I'm talking about.

@noroadsleft Will you approve this PR? What a need to do?

**Comment by noroadsleft** [comment]
I found an issue yesterday with the existing `info.json` file while working on something else entirely – not your fault, @luizcorreia – so I need to go through the `info.json` file anyway to check it.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**